app/src/main/java/com/moneytransfer/demo/MainActivity.kt

```diff
@@ -7,6 +7,7 @@ import androidx.navigation.findNavController
 import androidx.navigation.ui.AppBarConfiguration
 import androidx.navigation.ui.setupActionBarWithNavController
 import androidx.navigation.ui.setupWithNavController
+import com.moneytransfer.core.SampleCore
```

Comment:
remove this, we are not using this import

Reply:
Noted.